build/common-config.sh

```diff
@@ -35,7 +35,7 @@ var_set() {
 	"
 	env_key_sort[$i]="TINY_SCHEME_SRC"; i=$i+1; export TINY_SCHEME_SRC=${env_val[TINY_SCHEME_SRC]};
 
-	env_val[WASM_SRC]="${WASM_SRC:-${PDO_SOURCE_ROOT}/wasm}"
+	env_val[WASM_SRC]="${WASM_SRC:-${PDO_SOURCE_ROOT}/interpreters/wasm-micro-runtime}"
 	env_desc[WASM_SRC]="
```

Comment:
Given that this is getting checked out every time to ${PDO_SOURCE_ROOT}/interpreters/wasm-micro-runtime}, bash variable for WASM_SRC might be confusing. May I suggest that we hard code WASM_SRC to ${PDO_SOURCE_ROOT}/interpreters/wasm-micro-runtime}?

Reply:
By hard code the value of `WASM_SRC`, you mean, remove the `env_val[WASM_SRC]=` portion?

Reply:
avoid the optional WASM_SRC bash variable. not a big deal (I had to unset my variable for building this current PR)

Reply:
i would prefer to keep it. it still allows for experimentation with alternate branches/tags.